Spokestack/Trace.swift

```diff
@@ -33,12 +33,12 @@ public struct Trace {
         }
     }
     
-    /// Traces a debugging message.
-    /// - Parameter level: The debugging trace level for this message.
-    /// - Parameter configLevel: The speech pipeline's configured debugging trace level.
+    /// Traces a message from a Spokestack module.
+    /// - Parameter level: The trace level for this message.
+    /// - Parameter configLevel: The speech pipeline's configured trace level.
```

Comment:
You're passing a full `config` here, though.

Reply:
oh right, that's why i'm passing the full config. Needs to know the async UI-safe delegate. So the other `trace` arity keeps a similar signature.

Reply:
I just meant maybe the docstring here should change to reflect that.